src/sentry/middleware/integrations/parsers/jira.py

```diff
@@ -73,7 +74,11 @@ def get_response(self):
             )
 
         if self.view_class in self.immediate_response_region_classes:
-            return self.get_response_from_region_silo(region=regions[0])
+            try:
+                return self.get_response_from_region_silo(region=regions[0])
+            except ApiError as err:
+                sentry_sdk.capture_exception(err)
+                return self.get_response_from_control_silo()
```

Comment:
Isn't this covered in line 63? I guess same as my previous question.

Reply:
If we weren't able to get a response from the region due to timeouts/retry attempts being exceeded, I didn't want to let the exception bubble up as the response would be a 500. We could raise another an Http Exception that won't be captured as an error here as well.